app/services/notifications/index.service.js

```diff
@@ -0,0 +1,136 @@
+'use strict'
+
+/**
+ * Orchestrates presenting the data for `/notifications` page
+ * @module ViewNotificationsService
+ */
+
+const FetchEventNotificationsService = require('./fetch-events-notifications.service.js')
+const NotificationsIndexPresenter = require('../../presenters/notifications/index.presenter.js')
+const NotificationsIndexValidator = require('../../validators/notifications/index.validator.js')
+
+/**
+ * Orchestrates presenting the data for `/notifications` page
+ *
+ * @param {object} yar - The Hapi `request.yar` session manager passed on by the controller
+ *
+ * @returns {Promise<object>} The view data for the notifications page
+ */
+async function go(yar) {
+  const filters = await _getFilters(yar)
+
+  const { filterNotificationTypes } = filters
+
+  const _filterNotificationTypes = filters.filterNotificationTypes
+    ? _prepareNotifications(filterNotificationTypes)
+    : filterNotificationTypes
+
+  const validateResult = _validate(filters)
+
+  const filter = {
+    notifications: _filterNotificationTypes,
+    ...filters
+  }
```

Comment:
TBH, I had to play with this section a bit before I understood what it was doing.

I suggest moving this into a function so we can give it a name to help our future selves remember what is going on! Perhaps, `_determineSelectedNotificationTypes()` and `_prepareNotifications()` becomes `_convertTypesToBooleanFlags()`? 🤷